wagtail/admin/templates/wagtailadmin/workflows/confirm_delete.html

```diff
@@ -6,12 +6,12 @@
     {% include "wagtailadmin/shared/header.html" with title=view.page_title subtitle=view.get_page_subtitle icon=view.header_icon %}
 
     <div class="nice-padding">
-        <p>This workflow is currently in progress on {{ workflow_states_in_progress }} pages. Deleting this workflow will cancel moderation on these page.
-            Are you sure you want to delete this workflow?</p>
+        <p>{% blocktrans trimmed %}This workflow is currently in progress on {{ workflow_states_in_progress }} pages. Deleting this workflow will cancel moderation on these page.
+            Are you sure you want to delete this workflow? {% endblocktrans %}</p>
```

Comment:
```suggestion
            Are you sure you want to delete this workflow?{% endblocktrans %}</p>
```

One small space still ended up in this - suggested fix above.

Reply:
I think we probably also want a [`{% plural %}` tag](https://docs.djangoproject.com/en/4.0/topics/i18n/translation/#blocktranslate-template-tag) here, to account for the text reading "1 page" vs "2 pages".

Reply:
Here we need to use plural when the {{ workflow_states_in_progress }} >1 
how do i access the number of pages 
in the example its -
{% blocktrans count counter=list|length %}
can i use workflow_states_in_progress instead of list. I did search to see if i could find workflow_states_in_progress somewhere else in the code to understand its attributes better, can you help me understand this part. Thank you